packages/ui/src/components/icon/icon-name-map.ts

```diff
@@ -458,5 +460,7 @@ export const IconNameMap = {
   'node-icon': NodeIcon,
   'golang-icon': GolangIcon,
   'attachment-link': AttachmentLink,
-  spinner: Spinner
+  spinner: Spinner,
+  'close-new': CloseNew,
+  'tag-new': TagNew
```

Comment:
Shall we name it as `tag-2`? same for `close-new`.

Can we remove existing icons if not-needed/replaceable?